possible_topics.md

```diff
@@ -0,0 +1,81 @@
+# GraphQL Best Practices
+
+## Index
+
+- [1- Objective](#objective)
+- [2- Serving over http](#http)
+- [3- Main elements](#mainElements)
+  - [3.1- Queries](#queries)
+  - [3.2- Mutations](#mutations)
+  - [3.3- Types](#types)
+  - [3.4- Resolvers](#resolvers)
+  - [3.5- Field-resolvers](#fieldResolvers)
+  - [3.6- Subscriptions](#subscriptions)
+- [4- Nullability](#nullability)
+- [5- Pagination](#pagination)
+- [6- Caching and Batching](#cachingAndBatching)
+- [7- Authentication and Authorization](#auth)
+- [8- Error handling](#errorHandling)
+- [9- API versioning](#versioning)
+- [10- Apollo Server](#apolloServer)
+
+## Objective <a name="objective"></a>
+
+The purpose of this document is to present the conventions and standards used at Wolox for GraphQL. It is supposed to cover the main aspects of GraphQL from the Wolox's approach to it.
+
+## Serving over HTTP <a name="http"></a>
+
+Most modern web frameworks use a pipeline model where requests are passed through a stack of middleware. GraphQL should be placed after all authentication middleware, so that you have access to the same session and user information you would in your HTTP endpoint handlers.
+
+A standard GraphQL POST request should use the `application/json` content type, and include a JSON-encoded body of the following form:
```

Comment:
We could add some comments for the GET requests. Check this link: https://graphql.org/learn/serving-over-http/
We could add the link to the bibliography section I mentioned in this PR: https://github.com/lnmedrano/graphql-best-practices/pull/3